src/Notification/NotificationServiceProvider.php

```diff
@@ -30,33 +30,43 @@ public function boot()
      */
     public function registerNotificationTypes()
```

Comment:
this should be `protected` as well I think

Reply:
Yeah that makes sense to me